lib/checkother.h

```diff
@@ -312,7 +315,8 @@ class CPPCHECKLIB CheckOther : public Check {
                "- shadow variable.\n"
                "- variable can be declared const.\n"
                "- calculating modulo of one.\n"
-               "- known function argument, suspicious calculation.\n";
+               "- known function argument, suspicious calculation.\n"
+               "- suspicious comma in if/while condition statement.\n";
```

Comment:
suspicious comma immediately after function call in if/while condition statement.